SSO-Auth/Config/configPage.html

```diff
@@ -1,15 +1,272 @@
+              ></select>
+              <div class="selectArrowContainer">
+                <div style="visibility: hidden; display: none">0</div>
+                <span
+                  class="selectArrow material-icons keyboard_arrow_down"
+                  aria-hidden="true"
+                ></span>
+              </div>
             </div>
+
+            <button
+              id="LoadProvider"
+              is="emby-button"
+              type="button"
+              class="raised button-submit block emby-button"
+            >
+              <span>Load Provider</span>
+            </button>
+
+            <button
+              id="DeleteProvider"
+              is="emby-button"
+              type="button"
+              class="raised button-submit block emby-button"
+            >
+              <span>Delete Provider</span>
+            </button>
+          </form>
+
+          <form id="sso-new-oidc-provider" class="esqConfigurationForm">
+            <div class="verticalSection verticalSection-extrabottompadding">
+              <div class="inputContainer">
+                <label
+                  class="inputLabel inputLabelUnfocused"
+                  for="OidProviderName"
+                  >Name/ID of OID Provider:</label
+                >
+                <input
+                  is="emby-input"
+                  id="OidProviderName"
+                  required=""
+                  type="text"
+                />
+                <div class="fieldDescription">
+                  OID Provider Name to add / Update
+                </div>
+              </div>
+              <div class="inputContainer">
+                <label class="inputLabel inputLabelUnfocused" for="OidEndpoint"
+                  >OID Endpoint:</label
+                >
+                <input
+                  is="emby-input"
+                  id="OidEndpoint"
+                  required=""
+                  type="text"
+                />
+                <div class="fieldDescription">
+                  The OpenID endpoint. Must have a .well-known path available.
+                </div>
+              </div>
+
+              <div class="inputContainer">
+                <label class="inputLabel inputLabelUnfocused" for="OidClientId"
+                  >OpenID client ID:</label
+                >
+                <input
+                  is="emby-input"
+                  id="OidClientId"
+                  required=""
+                  type="text"
+                />
+                <div class="fieldDescription">The OpenID client ID.</div>
+              </div>
+              <div class="inputContainer">
+                <label class="inputLabel inputLabelUnfocused" for="OidSecret"
+                  >OID Secret:</label
+                >
+                <input is="emby-input" id="OidSecret" required="" type="text" />
+                <div class="fieldDescription">
+                  The OpenID Secret. Randomly Generated & Shared.
```

Comment:
Shouldn't be capitalized like this if it's a sentence. "The OpenID secret. Randomly generated and shared."

Reply:
https://github.com/9p4/jellyfin-plugin-sso/pull/18/commits/3bc8cc9a85dd51bd1f18dfb597b16a99d6197a2f#diff-d1cae9c0a00116bf22523395cbfe6b7f5d9603ef680a7202c05e4a53bc3b05cfL169-R173 resolved